src/models/hashmodel.h

```diff
@@ -104,6 +106,16 @@ class HashModel : public QAbstractTableModel
         return index(row, column);
     }
 
+    QModelIndex indexForSymbol(Data::Symbol symbol)
```

Comment:
Did you mean I should move it to CallerCalleeModel? I don't call indexForSymbol on the CostModel, at least not at this time